cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -102,12 +102,26 @@ public void createCollection(String dbName, String collectionName, long dataExpi
         BasicDBObject keys;
```

Comment:
An entry in CHANGES_NEXT_RELEASE file should be included, describing the changes in this PR. Something like this:

```
[cygnus-common] MongoDB indexes are created depending on DM (#2204)
```

Reply:
Added in 4ff6d9b